src/Request/Eshops/EshopOrdersBulk.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace SmartEmailing\v3\Request\Eshops;
+
+use SmartEmailing\v3\Api;
+use SmartEmailing\v3\Request\AbstractRequest;
+use SmartEmailing\v3\Request\Eshops\Model\Order;
+use SmartEmailing\v3\Request\Response;
+
+/**
+ * Class OrdersBulk
+ * Method is used to import orders in bulk. Up to 500 orders per request is allowed.
+ * This creates new activities which can be used for scoring, segmentation and automation.
+ *
+ * @package SmartEmailing\v3\Request\Eshops
+ */
+class EshopOrdersBulk extends AbstractRequest implements \JsonSerializable
```

Comment:
EshopOrdersBulk by bylo vhodnější implementovat bez AbstractRequest, toto funguje, ale nepřipadá mě to moc čisté  - hlavně tato implementace bude hodně náchylná tím, že se se přeimplementovává send